### PR TITLE
Update to newest modulesync_configs [3610af2]

### PR DESCRIPTION
**Pull request by DavidS**

### CODE REVIEW

**Comment by DavidS** [comment]
This requires a successful merge of https://github.com/puppetlabs/puppetlabs-xinetd/pull/72 before it can pass travis

File: Gemfile [hunk 1]
```diff
@@ -1,30 +1,39 @@
...
 
-group :development, :unit_tests do
-  gem 'rake',                    :require => false
-  gem 'rspec-puppet',            :require => false
-  gem 'puppetlabs_spec_helper',  :require => false
-  gem 'puppet-lint',             :require => false
-  gem 'simplecov',               :require => false
-  gem 'puppet_facts',            :require => false
-  gem 'json',                    :require => false
+def location_for(place, version = nil)
```
Comment (gregohardy) [inline Gemfile]: Should we source this location_for ?

Comment (DavidS) [inline Gemfile]: it's coming from modulesync_configs; there is no real libraries for Gemfiles: Catch-22

**Comment by gregohardy** [comment]
+1 I'll speak to you tomorrow about the location_for

File: Gemfile (hunk 1 above)
Comment (gregohardy) [inline Gemfile]: Coolio

**Pull request merged**